archive/b/brainfuck/README.md

```diff
@@ -6,6 +6,10 @@ Welcome to Sample Programs in Brainfuck!
 
 - [Hello World in Brainfuck](https://github.com/jrg94/sample-programs/issues/75)
 - [Reverse a String in Brainfuck](https://github.com/TheRenegadeCoder/sample-programs/issues/362)
+- [Fizz Buzz in Brainfuck](https://github.com/TheRenegadeCoder/sample-programs/issues/365)
```

Comment:
Nice job on the citations. These will go a long way in the future as we grow the repo.